### ISSUE TITLE
2 bugs with Move Down button (in Global Actions) 

### ISSUE THREAD

**Issue by xan2622** [InamuraJIN/ActionRecorder#20]
Blender version: 2.91
Add-on version: 3.5.6

_bug 1:_
In the "Global Actions - Edit Mode" category, if you move down the fist button (Merge), when this button reaches the end of the "Edit Mode" category, a "Delete" button appears.

_bug 2:_
And if you continue to click on the "Move Down" button, you can notice that it continues to move down in an "invisible category", and the button re-appears in the "UV Mode" Category but as "Merge". 

When you continue to click on "Move Down", when the "Merge" button reaches the end of the "External" category, it stops moving correctly.

![QcYU5tT1Ft](https://user-images.githubusercontent.com/1732773/103149641-71211b80-476b-11eb-8792-cfc3feb19b2b.gif)

To sum up: 
- When the button reaches the end of the "Edit Mode" category, the "Merge" button is renamed "Delete" (where does this Action come from? It wasn't there in the Global Actions before..
- Between the "Edit Mode" and "UV Mode" categories, there seems to be an invisible category

**Comment by RivinHD** [comment]
Ok, the bug is fixed. You will see the change with the next update.

**Comment by RivinHD** [comment]
Fixed with [Release 4.0.2](https://github.com/InamuraJIN/ActionRecorder/releases/tag/4.0.2)
